src/main/java/cinema/util/HashUtil.java

```diff
@@ -7,14 +7,17 @@
 public class HashUtil {
     public static final String HASH_ALGORITHM = "SHA-512";
 
-    public static String hashPassword(String password, byte[] salt) {
+    private HashUtil(){
+    }
+
+    public static String hashedPassword(String password, byte[] salt) {
         StringBuilder hashedPwd = new StringBuilder();
```

Comment:
i ment not the method's name but the name of a string builder lets fix it
```suggestion
    public static String hashPassword(String password, byte[] salt) {
        StringBuilder hashedPassword = new StringBuilder();
```